### DISCUSSION TITLE
Is it possible to put items to bottom of a card?

### DISCUSSION THREAD

**Discussion by Far0n** [h2oai/wave#1641] [Q&A] [answered]
Is it possible to put buttons (or any other item) on a `FormCard `like the following at the bottom page the page?

![grafik](https://user-images.githubusercontent.com/6949295/192725339-1b6b7484-8af8-44f8-b873-b03b1d20a9b4.png)

**Comment by mturoci** [comment, marked as answer]
Currently no, but I was thinking about a container like `ui.inline` but for vertical direction (something like `ui.column` for a. form card) which would allow that. Feel free to file a feature request!

**Reply by Far0n** [reply]
will do! thx
